lib/surface/components/dynamic_component.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Surface.Components.Component do
```

Comment:
Going by the file name:

```suggestion
defmodule Surface.Components.DynamicComponent do
```

Reply:
But in the tests it seems you named it `Component` on purpose, it does read nicer 🤔